From 409a1c4bb5b3468a72145ecad9253c22566a0e78 Mon Sep 17 00:00:00 2001
From: Mike Mueller <mike@subfocal.net>
Date: Fri, 17 Oct 2014 00:45:15 -0700
Subject: [PATCH] windows: Initialize mouse focus checkbox

Was showing unchecked, even if the mouse focus setting was set to 'sloppy'.
---
 capplets/windows/mate-window-properties.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/capplets/windows/mate-window-properties.c b/capplets/windows/mate-window-properties.c
index 03a9863..d2f3e4d 100644
--- a/capplets/windows/mate-window-properties.c
+++ b/capplets/windows/mate-window-properties.c
@@ -477,6 +477,8 @@ main (int argc, char **argv)
                       G_CALLBACK (mouse_focus_toggled_callback), NULL);
     g_signal_connect (marco_settings, "changed::" MARCO_FOCUS_KEY,
                       G_CALLBACK (mouse_focus_changed_callback), NULL);
+    /* Initialize the checkbox state appropriately */
+    mouse_focus_changed_callback(marco_settings, MARCO_FOCUS_KEY, NULL);
 
     g_settings_bind (marco_settings,
                      MARCO_AUTORAISE_KEY,
